src/SymbolicRegression.jl

```diff
@@ -24,6 +24,17 @@ export Population,
     simplifyWithSymbolicUtils,
     combineOperators,
     genRandomTree,
+    mutateConstant,
```

Comment:
Do these need to be exported? You can manually use them with:

```
using SymbolicRegression
...
SymbolicRegression.mutateConstant(...)
```

Exporting stuff is mainly for functions that a user would commonly use, rather than internal functions

Reply:
I tried not exporting those functions I added then use what you suggested, but it seems I cannot find them during runtime. I need access to all most of the low level functions since I am calling those functions in another directory, just like how one would call a library. Please feel free to remove these added exports if it doesn't match the package's general use cases.

Reply:
If you are experimenting, then I would just leave it as is, but not commit the code. You can add changes selectively with `git add -p`. If code is inside the package it should be able to access them with `@from ...`.

Reply:
Actually I think it should work, if you just call `@from ...` inside SymbolicRegression.jl. No need to export. You should be able to do `SymbolicRegression.mutateConstant`.

Reply:
Yeah it works, it just doesn't tab complete so I thought it is not there